dll/win32/mpr/wnet.c

```diff
@@ -2778,10 +2778,14 @@ DWORD WINAPI WNetConnectionDialog1W( LPCONNECTDLGSTRUCTW lpConnDlgStruct )
  */
 DWORD WINAPI WNetDisconnectDialog( HWND hwnd, DWORD dwType )
 {
-    FIXME( "(%p, %08X): stub\n", hwnd, dwType );
+    DISCDLGSTRUCTW disconn_dlg;
 
-    SetLastError(WN_NO_NETWORK);
-    return WN_NO_NETWORK;
+    ZeroMemory(&disconn_dlg, sizeof(disconn_dlg));
+	
+    disconn_dlg.cbStructure = sizeof(disconn_dlg);
+	disconn_dlg.hwndOwner = hwnd;
+	
+	return WNetDisconnectDialog1W(&disconn_dlg);
```

Comment:
Please surround these changes with #ifdef __REACTOS__, that's Wine code.

Reply:
Hopefully my lastest commit for MPR addresses it.